docs/manual/registration/objects-api.rst

```diff
@@ -0,0 +1,174 @@
+.. _manual_registration_objects_api:
+
+============
+Objecten API
+============
+
+Met de Objecten API-plugin kunnen inzendingen als "object" in een externe datalaag
+duurzaam geregistreerd worden. Deze data is vervolgens beschikbaar in het
+behandelproces.
+
+.. note:: De functioneel beheerder dient een aantal
+   :ref:`koppelingen <configuration_registration_objects>` in te stellen om deze plugin
+   te kunnen gebruiken. Let ook op de
+   :ref:`adviezen (Engels) <configuration_registration_objects_objecttype_tips>` voor
+   objecttypedefinities.
+
+Er zijn twee mogelijke manieren om de Objecten API in te zetten:
+
+* :ref:`manual_registration_objects_api_variables` (aangeraden)
+* :ref:`manual_registration_objects_api_templates` (verouderd)
+
+Deze worden hieronder toegelicht.
+
+.. _manual_registration_objects_api_variables:
+
+Variabelekoppelingen
+====================
+
+.. versionadded:: 2.6.0
```

Comment:
This produces text in english :thinking:

Reply:
The proper way to do this documentation is actually to use .po files for translation, but it's too labour intensive. I prefer the structured documentation (and the option to override the templates/styling for these blocks!) over consistent language.